src/main/java/com/ibm/watson/modelmesh/Metrics.java

```diff
@@ -230,6 +229,33 @@ public PrometheusMetrics(Map<String, String> params) throws Exception {
                 }
             }
 
+            if (!infoMetricParams.isEmpty()){
+                if (infoMetricParams.size() > INFO_METRICS_MAX) {
+                    throw new Exception("Too many info metrics provided in env var " + MMESH_CUSTOM_ENV_VAR + ": \""
+                            + infoMetricParams+ "\". The max is " + INFO_METRICS_MAX);
+                }
+
+                // Remove unset (unresolved) labels
+                for(String key : infoMetricParams.keySet()) {
+                    if(infoMetricParams.get(key).contains("$(")){
```

Comment:
I don't understand this logic... what is the `$(`? and why are we removing parameters that contain it?

Reply:
Removed this section, it was my own misunderstanding.